Library/Homebrew/test/spec_helper.rb

```diff
@@ -66,6 +66,8 @@
   config.order = :random
 
   config.raise_errors_for_deprecations!
+  config.warnings = true
+  config.disable_monkey_patching!
```

Comment:
Can we keep the “expose DSL” setting while disabling all other monkey patching? I don't really see the top-level `describe` being a problem since we don't ever call any top-level methods.

Adding 

```ruby
    mocks.syntax = :expect
    mocks.patch_marshal_to_support_partial_doubles = false
```

and 

```ruby
    expectations.syntax = :expect
```

below should do it, according to [https://rubydoc.info/github/rspec/rspec-core/RSpec%2FCore%2FConfiguration:disable_monkey_patching!](https://rubydoc.info/github/rspec/rspec-core/RSpec%2FCore%2FConfiguration:disable_monkey_patching!).

Reply:
> I don't really see the top-level `describe` being a problem since we don't ever call any top-level methods.

I like the explicitness of avoiding it.

Reply:
Also, many potential contributors will not be versed in the Ruby ecosystem, and this should make it clearer what test framework they need to include in their searches. 😄

Reply:
I am looking forward to the many potential contributors that will actually write a test. 😄